cmd/kubeadm/app/cmd/init.go

```diff
@@ -51,8 +51,7 @@ var (
 		To start using your cluster, you need to run (as a regular user):
 
 		  mkdir -p $HOME/.kube
-		  sudo cp -i {{.KubeConfigPath}} $HOME/.kube/config
-		  sudo chown $(id -u):$(id -g) $HOME/.kube/config
+		  sudo cat {{.KubeConfigPath}} > $HOME/.kube/config
```

Comment:
I'm worried that this breaks the wall of safety we had with `cp -i`... this will just overwrite whatever's there